_content/devices/node/quick-start.md

```diff
@@ -151,7 +151,7 @@ As you can see, the example sends all sensor data from the `interval()` callback
 ## Monitor & Decode Messages
 Let's see the messages come in.
 
-1.  From the application's screen in the console, select **Data** from the top right menu.
+1.  Since we are in the device overview we need to return to the application's screen in the console. On the top left breadcrumb click on your application name. Select **Data** from the top right menu.
```

Comment:
Can you add some punctuation, to make it more pleasant to read? Such as commas (e.g. *Since we are in the device overview, we need to return...*).

Reply:
@dotpy3 Sorry I am not a native English writer, so don't know where to exactly put them. I think the documentation is better to find the correct screen, how ever feel free to add the punctuation where you think it is usual. Thanks.